Chapter 1 Task 5

DeaDMop03 opened the pull request.

DeaDMop03 committed:
chapter1task5 release

ekolokolov commented on src/main/java/com/walking/intensive/chapter1/task5/Task5.java:
```diff
@@ -8,20 +10,33 @@ public static void main(String[] args) {
 //        Для собственных проверок можете делать любые изменения в этом методе
 
     }
+    static boolean checkTriangleExistance (double a, double b, double c) {
```
Тут явно какая то проблема. 
Для треугольника 12, 13, 5 - проверка не проходит.

ekolokolov reviewed (changes requested):
поправить проверку

DeaDMop03 committed:
chapter1task5 fixed condition on checkTriangleExistance

ekolokolov commented on src/main/java/com/walking/intensive/chapter1/task5/Task5.java:
```diff
@@ -48,19 +78,31 @@ static double[] getBisectors(double a, double b, double c) {
     static double[] getAngles(double a, double b, double c) {
```
Не корректно считает.

ekolokolov replied:
углы все еще не корректно считает

Для треугольника 12, 13, 5  примерно 22.619864, 67.3801350 и 90

ekolokolov commented on src/main/java/com/walking/intensive/chapter1/task5/Task5.java:
```diff
@@ -48,19 +78,31 @@ static double[] getBisectors(double a, double b, double c) {
+        return anglesArr;
     }
 
     static double getInscribedCircleRadius(double a, double b, double c) {
```
Не корректно считает

ekolokolov reviewed (changes requested):
Не корректное поведение на данных:
12, 13, 5

Expected :2.0
Actual   :0.5

DeaDMop03 committed:
chapter1task5
fixed getInscribedCircleRadius (semiPerimeter fault)
fixed anglesArr (denominator brackets fault)

EvgeneZhurov commented:
Висит более недели. Если требуется проверка - просьба открыть новый PR.
